cfme/tests/ansible/test_embedded_ansible_services.py

```diff
@@ -39,6 +42,69 @@
 ]
 
 
+@pytest.fixture()
+def ansible_linked_vm_action(
+    appliance, ansible_catalog_item, new_vm
+):
+    with update(ansible_catalog_item):
+        ansible_catalog_item.provisioning = {"playbook": "add_single_vm_to_service.yml"}
+
+    action_collection = appliance.collections.actions
```

Comment:
optional:
please use it directly in create.

Reply:
Updated as per directed.